wiki/Beatmaps/en.md

```diff
@@ -1,207 +1,72 @@
+Loved beatmaps use the heart icon (![Heart icon](/wiki/shared/status/loved.png)) in the song select screen.
 
-#### Beatmaps packs
+These beatmaps may not meet the [ranking criteria](/wiki/ranking_criteria) but are _loved_ by the community and thus are regularly added based on community votes. Before the beatmap begins playing, a dialoge will display, noting the status of the beatmap will cause:
 
-Beatmaps packs, as the title suggests, are a .zip file containing beatmaps. The name of the pack is based on what they contain (for example, "Approved Beatmap Pack \#7" would only contain Approved Beatmaps). It is worth noting that the most packs are:
+- No [pp](/wiki/pp) will be awarded.
+- All scores will be deleted when the creator updates the beatmap.
 
--   Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved).
--   Always contain videos (if the Beatmap has one).
--   Requires a program to open the .zip pack (like WinRAR, 7zip to name a few)
--   The beatmaps are either in the form of .zip or .osz file
+This status will give these beatmaps a leaderboard.
 
-The files are hosted on an external file storage website, so you don't have to wait for the cool down you get if you download too many beatmaps within a short period of time at osu! main website.
+To get a beatmap in the loved status, see the forum post: [Get your beatmap Loved!](https://osu.ppy.sh/forum/t/549835).
 
-Beatmaps packs split into 4 categories:
+### Pending
 
-| Category | Beatmaps type |
-| -------- | ------------- |
-| Standard | Ranked or Approved songs |
-| Chart | Fortnightly Chart challenge songs |
-| Theme | General theme of the songs |
-| Artist/Album | Based on Artist/Album |
+_See also: [Beatmap Ranking Procedure](/wiki/Beatmap_Ranking_Procedure)_
 
-If you really want **to fully complete your song list** (not just downloading your favorite songs only), **download all the (-/Approved/Taiko/osu!mania) Beatmaps Packs** in the **Standard** category and nothing else. The (-/Approved/Taiko/osu!mania) Beatmaps Packs will contain all the songs in the other 3 categories.
+Pending beatmaps use the question mark icon (![Question mark icon](/wiki/shared/status/pending.png)) in the song select screen.
 
-The Beatmaps packs can be found and downloaded from:-
+A beatmap with a pending status is usually waiting to be [modded](/wiki/modded). [Creators](/wiki/Creators) must seek for [modders](/wiki/modders) from the community and at least two [Beatmap Nominators](/wiki/Beatmap_Nominators) to get their beatmapset out of this status.
```

Comment:
a gist of what ppy wrote, they are called "creators"; do not use "mappers" anymore